randovania/games/super_metroid/patcher/super_duper_metroid_patcher.py

```diff
@@ -151,15 +185,44 @@ def create_patch_data(self, description: LayoutDescription, players_config: Play
             visual_etm=pickup_creator.create_visual_etm(),
         )
 
+        specific_patch_list = [attr for attr in vars(configuration.patches) if
+                               not callable(getattr(configuration.patches, attr)) and not attr.startswith("__")]
```

Comment:
I don't really follow what this code block is doing. Creating a list of all fields in `SuperPatchConfiguration`? In this case, you can use `dataclasses.fields`.

Reply:
I did not know this, that is much more convenient